src/http/http_server.cpp

```diff
@@ -57,6 +57,16 @@ error_s update_config(const http_request &req)
     return update_flag(iter->first, iter->second);
 }
 
+// If sub_path is 'app/duplication', the built path would be '<root_path>/app/duplication'.
+std::string build_rel_path(const std::string &root_path, const std::string &sub_path)
+{
+    std::string rel_path(root_path);
+    if (!rel_path.empty()) {
```

Comment:
In which case the root_path is empty?

Reply:
Root path of the metrics http service  is empty.